test/Identity.IntegrationTest/Endpoints/IdentityServerTests.cs

```diff
@@ -179,6 +178,255 @@ public async Task TokenEndpoint_GrantTypePassword_WrongAuthEmailHeader_Fails()
         AssertHelper.AssertJsonProperty(root, "error_description", JsonValueKind.String);
     }
 
+    [Theory]
+    [BitAutoData(OrganizationUserType.Owner)]
+    [BitAutoData(OrganizationUserType.Admin)]
+    public async Task TokenEndpoint_GrantTypePassword_WithOwnerOrAdmin_WithSsoPolicyEnabled_WithEnforceSsoPolicyForAllUsersTrue_Throw(OrganizationUserType organizationUserType, int generatedUsername)
```

Comment:
I think you might be able to combine some tests here.

For example, the assertions in this test applies to all users. You can combine:

`TokenEndpoint_GrantTypePassword_WithOwnerOrAdmin_WithSsoPolicyEnabled_WithEnforceSsoPolicyForAllUsersTrue_Throw`

and

`TokenEndpoint_GrantTypePassword_WithNonOwnerOrAdmin_WithSsoPolicyEnabled_WithEnforceSsoPolicyForAllUsersTrue_Throws`

Because the only difference between them is the `BitAutoData`, not any of the actual test code. _Everyone_ gets rejected by the server in this case.